blueprints/genome_search.py

```diff
@@ -160,6 +160,7 @@ def _prepare_response(self, grouped_genomes, total_matches):
                     common_name=genome['genome_info']['common_name'],
                     scientific_name=genome['genome_info']['scientific_name'],
                     assembly_name=genome['genome_info']['assembly_name'],
+                    url_slug=genome['genome_info']['url_slug'],
```

Comment:
Wouldn't it be better if we handle possible KeyError or using .get() to access values in python dictionaries?